chart_updater/updater.py

```diff
@@ -42,7 +42,8 @@ def _build_commit_message(manifest: Manifest) -> str:
 
     def _manifests_to_check(self) -> Iterator[str]:
         return (
-            path for path in self.git.grep(self.annotation_prefix + "/")
+            path for path in self.git.grep("HelmRelease")
+                if self.git.grep(self.annotation_prefix + "/", path)
```

Comment:
consider doing sets of "HelmRelease" and "self.annotation_prefix" greps and iterating only over their intersection

Reply:
OK, added.